charts/k8ssandra-test-cluster/templates/tls-certificate-bundle.yaml

```
@@ -0,0 +1,24 @@
+{{- if and .Values.client_encryption_options.enabled .Values.syncCACertToSecret }}
+# Let trust-manager sync the CA PEM (and only that!) into secrets named
+# `k8ssandra-tls-ca-certificate-<namespace>` in all configured namespaces or only
+# one if syncCACertNamespace is defined. This way we can hide the private key
+# from public.
+apiVersion: trust.cert-manager.io/v1alpha1
+kind: Bundle
+metadata:
+  name: k8ssandra-tls-ca-certificate-{{- .Release.Namespace }}
```

Nit: IMO adding name to names of already namespaced objects just creates noise when listing k8s objects.

```suggestion
  name: k8ssandra-tls-ca-certificate
```

---

`trust-manager` will create (sync) one or more secrets with exactly this name. I've added this to avoid collisions when multiple wire instances are installed in one cluster. However, we may force people to keep their `trust-manager`-story clean and not clutter unused secrets around. Finally, this is our test setup code provided as example to other users. So, it should be fine to make some assumptions. :+1: